mitodl/ocw-studio
# add models, admin config, and the migration for the WebsiteCollection

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #315

#### What's this PR do?

This adds two models, a `WebsiteCollection` and a `WebsiteCollectionItem`. The `WebsiteCollectionItem` is basically a join table between `WebsiteCollection` and `Website` which also has a `position` field - this will let us implement basically a simple array of `Websites`, similarly to how we did our `UserList` model for open-discussions. I didn't implement the stuff for adding, removing, and re-ordering `WebsiteCollections` because I think that code will end up be in our serializer.

#### How should this be manually tested?

The migration should run without issue, and you should be able to add and edit the models in the admin.
